### ISSUE TITLE
Emoji not displayed properly

### ISSUE THREAD

**Issue by abbyad** [medic/medic-conf#55]
...on Windows

```
C:\Users\Marc\Documents\GitHub\medic-projects\standard>medic-conf --instance [REDACTED] upload-app-forms
­ƒöæ  Password: ********************************
```

I haven't found a way to enable better character support (without changing OS), so anyone on Windows configuring our instances will encounter this.

**Comment by alxndrsn** [comment]
Does your console support emojis at all?

**Comment by alxndrsn** [comment]
Not planning to fix this, sorry 😢

**Comment by abbyad** [comment]
No, Windows does not yet support emoji in any of the shells. Given that we have Windows users for this tool we should remove emoji when Windows is detected, or not use emoji for this tool. The added benefit of seeing a key emoji is not worth falling back to `­ƒöæ`.

**Comment by alxndrsn** [comment]
Fixed in `1.11.3`.
